exporter.py

```diff
@@ -45,6 +44,40 @@ def collect_data(nsip, entity, username, password, protocol, nitro_timeout):
 
     # Login credentials
     headers = {'X-NITRO-USER': username, 'X-NITRO-PASS': password}
+    if (entity == 'lbvserver_binding'):
+        url_lbvserver = '%s://%s/nitro/v1/config/lbvserver' % (protocol, nsip)
+
+        responselbvserver = requests.get(url_lbvserver, headers=headers, verify=False, timeout=nitro_timeout)
+        rlbvserver = responselbvserver.json()
+
+        lbvserver_binding_status_up = {'lbvserver_binding': []}
+
+        for lbvserver in rlbvserver['lbvserver']:
+            url = '%s://%s/nitro/v1/config/lbvserver_binding/%s' % (protocol, nsip, lbvserver['name'])
+            r = requests.get(url, headers=headers, verify=False, timeout=nitro_timeout)
+            values = r.json()
+            total = 0
+            total_down = 0
+            total_up = 0
+            for lbvserver_binding in values['lbvserver_binding']:
+                if ('lbvserver_servicegroupmember_binding' in lbvserver_binding):
+                    for lbvserver_binding_servers in lbvserver_binding['lbvserver_servicegroupmember_binding']:
+                        total += 1
+                        if lbvserver_binding_servers['curstate'] == "UP":
+                            total_up += 1
+                        else:
+                            total_down += 1
+                if total != 0:
+                    percentup = (total_up/total) * 100
```

Comment:
This calculation of "percentup" does not seem to be correct for python2. as 1/3 will give 0. If we can make it compatible by adding float as denominator, then should be fine.